backend/api/v2beta1/pipeline.proto

```diff
@@ -125,10 +124,11 @@ service PipelineService {
   rpc ListPipelineVersions(ListPipelineVersionsRequest)
       returns (ListPipelineVersionsResponse) {
     option (google.api.http) = {
-      get: "/apis/v2beta1/pipeline_versions"
+      get: "/apis/v2beta1/pipelines/{pipeline_id}/pipeline_versions"
```

Comment:
line 108 needs to be updated too? 
```
- post: "/apis/v2beta1/pipeline_versions"
+ post: "/apis/v2beta1/pipelines/{pipeline_id}/pipeline_versions"
```

Reply:
Also I believe it was `/versions` not `/pipeline_versions` since pipeline part is inferred

Reply:
Agree , Thank you, @annajung